lib/web_ui/lib/src/engine/text_editing.dart

```diff
@@ -341,6 +341,7 @@ class TextEditingElement {
       _subscriptions.add(domElement.onKeyUp.listen((event) {
         _handleChange(event);
       }));
+      _subscriptions.add(domElement.onSelect.listen(_handleChange));
```

Comment:
Could you add a comment explaining why this is necessary?

Reply:
Done.